rules/no-watch/no-watch.ts.test.js

```diff
@@ -0,0 +1,105 @@
+const { RuleTester } =
+  require("@typescript-eslint/experimental-utils").ESLintUtils;
+const { join } = require("path");
+
+const { readExample } = require("../../utils/read-example");
+const rule = require("./no-watch");
+
+const ruleTester = new RuleTester({
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: "module",
+    project: "./tsconfig.json",
+    tsconfigRootDir: join(__dirname, ".."),
+  },
+});
+
+const readExampleForTheRule = (name) => ({
+  code: readExample(__dirname, name),
+  filename: join(__dirname, "examples", name),
+});
+
+ruleTester.run("effector/no-watch.ts.test", rule, {
+  valid: ["correct.ts"].map(readExampleForTheRule),
```

Comment:
It's still noting for correct cases like `forward({...})`, factories, etc. 🥺

Reply:
What do you mean? I added cases for guard, sample, effects and etc inside correct.ts